doc/Makefile

```diff
@@ -32,14 +32,17 @@ cleanall: clean
 html: deps
 	@echo "Building HTML documentation."
 ifneq ($(OS),WINNT)
-	$(JULIA_EXECUTABLE) $(call cygpath_w,$(SRCDIR)/make.jl) -- deploy
+	$(JULIA_EXECUTABLE) $(call cygpath_w,$(SRCDIR)/make.jl) -- $(ARGS)
 else
 # work around issue #11727, windows output redirection breaking on buildbot
-	$(JULIA_EXECUTABLE) $(call cygpath_w,$(SRCDIR)/make.jl) -- deploy > docbuild.log 2>&1
+	$(JULIA_EXECUTABLE) $(call cygpath_w,$(SRCDIR)/make.jl) -- $(ARGS) > docbuild.log 2>&1
 	@cat docbuild.log
 endif
 	@echo "Build finished. The HTML pages are in _build/html."
 
+deploy: html
+deploy: ARGS := deploy
```

Comment:
Yup. The command that gets executed in my case is

```
/home/morten/Julia/julia/build/usr/bin/julia /home/morten/Julia/julia/doc/make.jl --
```

It might be a good idea to explicitly set it in the Makefile though, to make that clear.